src/Apps/Order/Components/__stories__/ItemReview.story.tsx

```diff
@@ -10,48 +11,76 @@ storiesOf("Styleguide/Components", module).add("ItemReview", () => {
       <Section title="Item Review">
         <Flex width={542} flexDirection="column">
           <ItemReview
-            artist="Francesca DiMattio"
-            title="The Fox and the Hound"
-            date="2018"
-            medium="Oil on canvas"
-            dimensions={{
-              in: "96 × 79 in",
-              cm: "243.8 × 200.7 cm",
+            artwork={{
+              " $refType": Symbol() as ItemReview_artwork$ref,
```

Comment:
You can also pass `null` without casting and the value is never actually used.

Reply:
done! :+1: